src/Core/Users/class.user.php

```diff
@@ -0,0 +1,77 @@
+
+<?php
+namespace Meetup\Login\Core;
+
+use Meetup\Login\Core\Users\UserInterface
+
+Class User implements UserInterface 
+{
+
+	public function __construct($email, $password){
+		$this->setEmail($email);
+		$this->setPassword($password);
+	}
+
+	public function setPassword($password){
+		$this->password = $password;
+	}
+
+	public function setEmail($email){
+		$this->email = $email;
+	}
+
+	public function checkPassword(){
+		$sql = "SELECT * FROM user WHERE $email='".$this->email."' AND password='".AES_DECRYPT($this->password, 'meetup')."' ";
+		$query = mysql_query($sql);
+		if(mysql_num_rows() > 0)
+			return mysql_fetch_row($query);
+		else
+			return false;
+	}
+	
+	public function setFirstName($firstName){
+		$this->firstName = $firstName;
+	}
+	public function getFirstName(){
+		return $this->firstName;
+	}
+	public function setLastName($lastName){
+		$this->lastName = $lastName;
+	}
+	public function getLastName(){
+		return $this->lastName;
+	}
+
+    public function getCreatedAt(){
+		return $this->createdAt;
+	}
+
+    public function setCreatedAt($createdTime){
+		$this->createdAt = $createdTime;
+	}
+
+    public function getUpdatedAt(){
+		return $this->updatedTime;
+	}
+
+    public function setUpdatedAt($datetime){
+		$this->updatedTime = $dateTime;
+		$sql = "SELECT id FROM user WHERE $email='".$this->email."' updated_at='".$datetime."' ";
```

Comment:
Why the query? We are trying to create an `User` entity. We are not concerned about persisting it, at the moment. Please remove all queries from methods.